packages/p2p-v2/src/utils/file-dropzone.ts

```diff
@@ -0,0 +1,14 @@
+import { ReactElement, ReactNode } from 'react';
+import { DropzoneOptions } from 'react-dropzone';
+
+export type TFileDropzone = DropzoneOptions & {
```

Comment:
shouldn't this be in types file?

Reply:
I think we need to sort out the types file, we have 2 😅 and the types are kind of merged all together, i think we should have a separate types folder, and then we have separate files for different types, like one for API types, component prop types, etc, but let me bring this topic in the channel so we can discuss with the others

Reply:
for now i'll leave this here, I've written in the channel about migrating these to either one file using namespaces or we can do folder based